src/Coven.Agents.OpenAI/OpenAIResponseOptionsTransmuter.cs

```diff
@@ -21,23 +21,29 @@ public Task<ResponseCreationOptions> Transmute(OpenAIClientConfig Input, Cancell
 
         if (Input.Reasoning is not null)
```

Comment:
This one is definitely my bad, Input.Reasoning should be nullable and default to null. Would you be willing to implement that direction?